webpack.config.client.js

```diff
@@ -7,6 +8,8 @@ const dotenv = require('dotenv');
 const { DuplicatesPlugin } = require('inspectpack/plugin');
 const { getClientEnvVars } = require('./src/clientEnvVars');
 
+const FRAMEWORK_BUNDLES = ['react', 'react-dom'];
+const TOTAL_PAGES = 13;
```

Comment:
any ideas how we automate this? I can only think of reading the pages directory with `fs`

Reply:
I can't think of another way? What's the concern with using `fs`?